src/backend/app/db/postgis_utils.py

```diff
@@ -785,3 +791,124 @@ def parse_featcol(features: Union[Feature, FeatCol, MultiPolygon, Polygon]):
     elif isinstance(features, Feature):
         feat_col = geojson.FeatureCollection([feat_col])
     return feat_col
+
+
+def get_osm_geometries(form_category, geometry):
+    """Request a snapshot based on the provided geometry.
+
+    Args:
+        form_category(str): feature category type (eg: buildings).
+        geometry (str): The geometry data in JSON format.
+
+    Returns:
+        dict: The JSON response containing the snapshot data.
+    """
+    config_filename = XLSFormType(form_category).name
+    data_model = f"{data_models_path}/{config_filename}.yaml"
+
+    with open(data_model, "rb") as data_model_yaml:
+        extract_config = BytesIO(data_model_yaml.read())
+
+    pg = PostgresClient(
+        "underpass",
+        extract_config,
+        auth_token=settings.RAW_DATA_API_AUTH_TOKEN
+        if settings.RAW_DATA_API_AUTH_TOKEN
+        else None,
+    )
+    return pg.execQuery(
+        geometry,
+        extra_params={
+            "fileName": (
+                f"fmtm/{settings.FMTM_DOMAIN}/data_extract"
+                if settings.RAW_DATA_API_AUTH_TOKEN
+                else "fmtm_extract"
+            ),
+            "outputType": "geojson",
```

Comment:
I believe bind zip true and geojson are defaults on raw data api, but no harm being explicit 👍

Reply:
I think its better to mention, as we won't know what are the default values.